source/develop/release-management/features/network/sriov-live-migration.md

```diff
@@ -0,0 +1,178 @@
+---
+title: SR-IOV Live Migration without downtime
+category: feature
+authors: amusil
+feature_name: SR-IOV Live Migration without downtime
+feature_modules: engine,network,vdsm
+feature_status: In Development
+---
+
+# SR-IOV Live Migration without downtime
+
+## Summary
+
+This feature allows you to migrate VM that is connected via SR-IOV with
+a minimal network downtime and supersedes the [Live Migration Support For SRIOV][1]
+that required additional configuration inside affected VM.
+
+The state of the feature is tracked in [[RFE] [SR-IOV] Migration should not require downtime as of today][2].
+Related patches can be found on [gerrit topic:sriov_live_migration][3].
+
+## Owner
+
+*   Name: [Ales Musil](https://github.com/almusil)
+
+*   Email: <amusil@redhat.com>
+
+
+## Detailed Description
+
+In order to migrate a VM that is connected to SR-IOV passthrough profile
+we need to first unplug the SR-IOV device from a VM on the source, migrate
+the VM and plug SR-IOV device on the destination. This causes longer downtime
+during which the VM is not connected to the network.
+
+The current implementation allowed to mitigate this downtime
+[Live Migration Support For SRIOV][1], however this approach required manual
+configuration to be done in the guest which is not convenient for large setups. 
+
+In order to achieve minimal downtime the [NET_FAILOVER][4] support was added 
+to Linux kernel. The driver provides automated mechanism which is transparent 
+to the VM, and the user space. It is achieved by `failover` netdev which acts
+as master device and controls two slave devices, the `primary` device being 
+SR-IOV passtrough and the virtio as `standby`. The VM user access network
+through the `failover` device. 
+
+There is still need to provide two interfaces, the SR-IOV passtrough device
+and virtio device, to the affected VM, but the  overall configuration overhead
+has been reduced to selecting suitable vNIC profile pair.
+
```

Comment:
Do we already know, if the virtio device can be connected to the same physical NIC as the VF?

Reply:
Maybe we can conserve this as an open question?

Reply:
There is a new section called open questions, which can be updated as we find the answers